src/protvista-uniprot-structure.ts

```diff
@@ -140,6 +149,15 @@ class ProtvistaUniprotStructure extends LitElement {
     protvistaDatatableElt.selectedid = this.pdbId;
   }
 
+  addStyles() {
+    // We are not using static get styles()
+    // as we are not using the shadowDOM
+    // because of Mol*
+    const styleTag = document.createElement('style');
+    styleTag.innerHTML = loaderStyles.toString();
+    document.querySelector('head')?.append(styleTag);
+  }
+
```

Comment:
This should probably cleaned up afterwards if added in the global scope (using connectedCallback and disconnectedCallback).
Either that, or be added once on module load

Reply:
Also, if the styles are not gonna be scoped in shadow DOM, the selectors should probably be more specific (`protvista-uniprot-structure .my-class` for example)

Reply:
Added logic to only add styles once and remove when disconnected. I'll make a Jira about the specificity of classes - as that was not changed.